Automattic/jetpack
# Add twitter sharing button

Fixes https://github.com/Automattic/jetpack/issues/35141

## Proposed changes:

Adding Twitter icon support for Sharing Buttons.

<img width="235" alt="Screenshot 2024-01-22 at 3 14 27 PM" src="https://github.com/Automattic/jetpack/assets/60262784/ebdb1f21-2fed-4046-a328-b7a52cab0b8d">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up your site with local changes
 - Make sure you have beta block enabled. In your environment (example Docker) wp-config.php you should have define( 'JETPACK_BLOCKS_VARIATION', 'beta' );
- Add Sharing Buttons block to a post.
- Add Twitter Block
